modules/custom_operations/CMakeLists.txt

```diff
@@ -13,5 +16,8 @@ find_package(OpenVINODeveloperPackage QUIET)
 if(NOT OpenVINODeveloperPackage_FOUND)
     find_package(OpenVINO REQUIRED COMPONENTS Runtime)
 endif()
+set(CMAKE_PROJECT_VERSION "${OpenVINO_VERSION}.${PROJECT_VERSION_PATCH}")
+
+message("${CMAKE_PROJECT_NAME} version is ${CMAKE_PROJECT_VERSION}")
```

Comment:
let's remove it, looks like a debug message :)

Reply:
done